unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/LoadingScreen/Scripts/LoadingScreenTipsController.cs

```diff
@@ -63,6 +63,9 @@ private async UniTask IterateTipsAsync()
 
         public void StopTips()
         {
+            if (!tipsView.gameObject.activeSelf) return;
+
+            tipsView.gameObject.SetActive(false);
             disposeCts.Cancel();
```

Comment:
```
disposeCts.Cancel();
disposeCts?.Dispose();
```

Checking for nullity on the second line makes no sense as NRE can occur on the first one.
Also considering how you use it I would recommend checking the token for nullity instead of `activeSelf` so you will get rid of `?` on both lines.

```
if (disposeCts ==  null) return;
```

Reply:
Makes sense. Done